server/service/src/vaccine_course/test/update.rs

```diff
@@ -39,6 +39,7 @@ mod query {
             coverage_rate: 100.0,
             use_in_gaps_calculations: true,
             wastage_rate: 0.1,
+            can_skip_dose: false,
```

Comment:
You can use `..Default::default()` in test cases to avoid the need to change all these files next time we want to add to vaccine_course